tests/integration-tests/tests/networking/test_cluster_networking.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import pytest
+from assertpy import assert_that
+
+
+@pytest.mark.dimensions("eu-west-1", "c4.xlarge", "alinux2", "slurm")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "alinux2", "sge")
+@pytest.mark.usefixtures("scheduler, os, region, test_datadir")
```

Comment:
why test_datadir in the usefixtures? can't it be removed?

you also need to add instance to the fixtures.

Reply:
Test_datadir is needed from pcluster_config_reader, isn't it?

Reply:
yes but pcluster_config will get the fixture regardless of it